src/models/tests/mock.ts

```diff
@@ -47,46 +39,7 @@ Object.defineProperty(window, "RTCPeerConnection", {
   })),
 });
 
-export class MockAudioContext extends AudioContext {
```

Comment:
This whole thing is not needed anymore because the updated `standardized-audio-context-mock` provides its own mock `StereoPannerNode` for us.